dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -270,9 +270,12 @@ def hoc_signups
   def retention
     require 'cdo/properties'
     SeamlessDatabasePool.use_persistent_read_connection do
-      @scripts = params[:scripts_ids].present? ?
-        params[:scripts_ids].split(',').map(&:to_i) :
-        [1, 17, 18, 19, 23]  # Default to the CSF scripts.
+      # If no scripts are specified, default to the CSF scripts.
+      if !params[:script_ids].present?
+        params[:script_ids] = '1, 17, 18, 19, 23'
```

Comment:
yes, I agree, what you had before (setting a variable based on the presence or not of a value in the params hash)... any particular reason you changed it?